free-programming-books-zh.md

```diff
@@ -86,6 +87,8 @@
 * [正则表达式30分钟入门教程](http://deerchao.net/tutorials/regex/regex.htm)
 
 
+### AWK
+* [AWK 程序设计语言](https://github.com/wuzhouhui/awk)
```

Comment:
https://en.wikipedia.org/wiki/The_AWK_Programming_Language isnt in the public domain, yes you are translating it but i have to defer to @vhf to decide whether we can include.